content/kb/troubleshooting-volume-filesystem-corruption.md

```diff
@@ -48,3 +51,36 @@ Longhorn cannot fix this automatically. You will need to resolve this manually w
 7. Detach the volume from the UI.
 8. Scale up the workload.
 
+### For some older Linux distribution versions and Longhorn volumes with ext4 filesystems
+
+In the CSI flow, the Longhorn CSI plugin creates a file system on a new volume using a `mkfs.ext4` built into its
```

Comment:
Probably should consider using the same version of `e2fsprogs` as the host instead of using the built-in in our image. The built-in one can be a fallback. WDYT? @ejweber